### DISCUSSION TITLE
integration downloading a significant amount of data

### DISCUSSION THREAD

**Discussion by m4n63** [hass-chargeamps/hass-chargeamps#70] [Q&A] [answered]
I noticed that this integration seem to download quite a bit of data. In my case around 190GB per month. This started for me around the 16th of September. Anyone else noticing the same behaviour?

**Comment by jschlyter** [comment, marked as answer]
You could try to lower the `scan_interval` from default 30 seconds to 300 seconds. Perhaps the data includes more information since mid-September? IIRC they did change the API at some point.

**Reply by m4n63** [reply]
Thank you for the suggeestion. The data being downloaded is not any real issue to me as such though, but I just figured it was an odd thing and a really large chunk of data for what I believed was just a few API calls. I was mainly interested in whether other people see the same behaviour, and if not maybe I should try to delete and reinstall the integration 🤔. 
After the change, perhaps it downloads a big chunk of data and parses values out locally or something...  oh well.
EDIT: tried reinstalling and same behaviour, so I suspect this is by design.
